### PR TITLE
4/1愚人節 金坷垃

### PR DESCRIPTION
**Pull request by jacky860226**
4/1全面金坷垃化
剩下換頁不能連續播放的問題

### CODE REVIEW

File: index/templates/index/base.html [hunk 1]
```diff
@@ -123,6 +130,19 @@
     <script src={% static "js/popup-setting.js" %}></script>
     <script src={% static "js/base.js" %}></script>
     <script src={% static "js/clock.js" %}></script>
+    <script>
+        var data=new Date();
+        var month=4-1,date=1;
```
Comment (bruce3557) [inline index/templates/index/base.html:135]: 4-1?

File: index/templates/index/index.html [hunk 2]
```diff
@@ -66,6 +85,18 @@ <h3 style="display: inline;"><b>Announcements</b></h3>
       </tbody>
     </table>
   </div>
+  <!-- js -->
+  <script>
+        var data=new Date();
+        var month=4-1,date=1;
```
Comment (bruce3557) [inline index/templates/index/index.html:91]: ditto

**Comment by bruce3557** [comment]
I love this plan XDDD
Happy to see it going to production

File: index/templates/index/index.html (hunk 2 above)
Comment (jacky860226) [inline index/templates/index/index.html:91]: 他月的編號是從0開始的

Comment (tony24681379) [inline index/templates/index/index.html:91]: http://www.victsao.com/blog/81-javascript/193-javascript-date-getmonth

Comment (bruce3557) [inline index/templates/index/index.html:91]: Fine

**Comment by bruce3557** [comment]
+1, going to merge

**Pull request merged**